components/elm/src/biogeochem/FireMod.F90

```diff
@@ -370,12 +371,14 @@ subroutine FireArea (bounds, &
 
               ! For non-crop -- natural vegetation and bare-soil
               if( veg_pp%itype(p)  <  nc3crop .and. cropf_col(c)  <  1.0_r8 )then
-                 if( .not. (btran2(p) .ne. btran2(p)))then !?shr_infnan_isnan(btran2(p))) then
-                    if (btran2(p)  <=  1._r8 ) then
-                       btran_col(c) = btran_col(c)+btran2(p)*veg_pp%wtcol(p)
-                       wtlf(c)      = wtlf(c)+veg_pp%wtcol(p)
+                 if(.not. shr_infnan_isnan(btran2(p)) ) then
```

Comment:
Why would `btran2(p)` be `nan` to begin with?

Reply:
It's`nan` due to a finidat file used for a test that has it as `nan`.

Reply:
How about fixing the finidat file of the test and leaving the code unchanged?